addons/actions/README.md

```diff
@@ -18,10 +18,43 @@ Then, add following content to `.storybook/main.js`
 
 ```js
 module.exports = {
-  addons: ['@storybook/addon-actions']
-}
+  addons: ['@storybook/addon-actions'],
+};
+```
+
+## Actions args
+
+Starting in SB6.0, we recommend using the `actions` story parameter to specify actions. There are two relevant options `actions.args` and `actions.argTypesRegex` which the actions addon uses to inject actions into your story functions as Storybook Args.
+
+The following example uses the `actions.args` array to generate actions that are passed into the story (when `passArgsFirst` is set to `true`):
+
+```js
+import Button from './button';
+
+export default {
+  title: 'Button',
+  parameters: { actions: { args: ['onClick'] } },
+};
+
+export const defaultView = ({ onClick }) => <Button onClick={onClick}>Hello World!</Button>;
 ```
 
+Alternatively, suppose you have a naming convention, like `onX` for event handlers. The following configuration automatically creates actions for each `onX` argType (which you can either specify manually or generate automatically using [Storybook Docs](https://www.npmjs.com/package/@storybook/addon-docs).
```

Comment:
Maybe add an example of how you'd provide an argType to the story? Or is that sort of pointless?

Reply:
Waiting on this until the argTypes design is finalized